apis/outlook/calendar/code/pkg/commands/listEventsToday.go

```diff
@@ -0,0 +1,56 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gptscript-ai/tools/apis/outlook/calendar/code/pkg/client"
+	"github.com/gptscript-ai/tools/apis/outlook/calendar/code/pkg/global"
+	"github.com/gptscript-ai/tools/apis/outlook/calendar/code/pkg/graph"
+	"github.com/gptscript-ai/tools/apis/outlook/calendar/code/pkg/printers"
+	"github.com/gptscript-ai/tools/apis/outlook/calendar/code/pkg/util"
+)
+
+func ListEventsToday(ctx context.Context) error {
+	c, err := client.NewClient(global.ReadOnlyScopes)
+	if err != nil {
+		return fmt.Errorf("failed to create client: %w", err)
+	}
+
+	calendars, err := graph.ListCalendars(ctx, c)
+	if err != nil {
+		return fmt.Errorf("failed to list calendars: %w", err)
+	}
+
+	for _, cal := range calendars {
+		if cal.ID == "" {
+			continue
+		}
+
+		events, err := graph.ListCalendarViewToday(ctx, c, cal.ID, cal.Owner)
```

Comment:
I see that this is different for `ListEvents` and `ListEventsToday`, but does it need to be?

Reply:
Fixed